src/Components/Common/HelperInputFields.tsx

```diff
@@ -96,7 +96,7 @@ export const TextInputField = (props: TextFieldPropsExtended) => {
             return
         }
         if (type === 'number' && event.target.value) {
-            event.target.value = event.target.value.replace(/\D/, '');
+            event.target.value = event.target.value.replace(/(?!\.)\D/, '');
```

Comment:
instead of doing it to numbers, you can create something like,
`if (type === 'float' && event.target.value) {
            event.target.value = event.target.value.replace(/(?!\.)\D/, '');
}
`
this way it wont effect other places where number is used

Reply:
Okay, will do it...